runtime-testsuite/test/org/antlr/v4/runtime/TestCodePointCharStream.java

```diff
@@ -3,17 +3,13 @@
  * Use of this file is governed by the BSD 3-clause license that
  * can be found in the LICENSE.txt file in the project root.
  */
-package org.antlr.v4.test.runtime.java;
+package org.antlr.v4.runtime;
```

Comment:
ok, we should definitely keep it in the testing things so it does not get pushed out as part of the runtime jar. Can we separate somehow?